nixos/modules/installer/cd-dvd/iso-image.nix

```
@@ -403,7 +421,9 @@ let
       mkdir ./boot
       cp -p "${config.boot.kernelPackages.kernel}/${config.system.boot.loader.kernelFile}" \
         "${config.system.build.initialRamdisk}/${config.system.boot.loader.initrdFile}" ./boot/
-      touch --date=@0 ./EFI ./boot
+
+      # Rewrite dates for everything in the FS
+      find . -exec touch --date=@$DATE '{}' ';'
```

```suggestion
      find . -exec touch --date=2000-01-01 {} +
```

to process more than one file at once